php/ubiquity/.htaccess

```diff
@@ -0,0 +1,10 @@
+AddDefaultCharset UTF-8
```

Comment:
can we remove this file ?

I mean this is for `apache` compatibility, right ?

Reply:
yes, we can!